src/main/fc/settings.yaml

```diff
@@ -1307,6 +1307,9 @@ groups:
       - name: nav_max_terrain_follow_alt
         field: general.max_terrain_follow_altitude
         max: 1000
+      - name: nav_max_altitude
+        field: general.max_altitude
+        max: 65000
```

Comment:
Better adding a sane lower limit? (also for the setting validation)

Reply:
650m is pretty sane to me :)

Reply:
Was referring to the minium. Like 100cm ? just to allow users to take off at least and not to set negative altitudes

Reply:
@DzikuVx More of a curiosity... Where did the `650m` max altitude value come from?  I'd understand a value of `120m` (albeit causing problems for many).  And not that I'd do this, but could this be set to `0` to disable?  It just seems a bit ambiguous to default to a `650m` ceiling, but I'm probably missing something.  I'd never fly that high anyway, and I live in a very flat part of the world, so it makes zero difference to me.

Reply:
uint16_t max value.
The default is 0, disabled.

Reply:
So how is it not impossible that they didn't fail to not notice this during any and every single test? Did they never enter nothing into none of those fields? Or didn't they release this wrapper library without testing nothing at all?